src/types/schema.ts

```diff
@@ -1,65 +1,108 @@
 import { UUID } from 'crypto';
 
-export interface CaseListing {
-  id: UUID;
-  title: string;
-  summary: string;
-  country: string;
-  client_location: string;
-  legal_server_id: number;
-  hours_per_month: number;
-  num_months: number;
-  languages: string[];
-  is_remote: boolean;
-  in_court: boolean;
-  needs_attorney: boolean;
-  needs_interpreter: boolean;
-  upcoming_date: string;
-  relief_sought: string;
-  active: boolean;
+// table field types
+export type ImmigrationLawExperienceEnum =
+  | 'No Experience'
```

Comment:
Can we change this to `LOW`, `MEDIUM`, `HIGH`? Sorry for the absurd number of changes to this, but David and I ultimately decided that these words would represent no prior cases, 1 prior case, and multiple prior cases, respectively. See this from interest form design:

<img width="335" alt="Screenshot 2023-10-31 at 8 41 45 PM" src="https://github.com/calblueprint/immigration-justice-project/assets/47801900/40ab4a6d-0eb3-411f-a807-d1de09c08cbb">

Reply:
Do we want one-word names for the other `enum`s as well?